### ISSUE TITLE
Provide DSL to configure store settings

### ISSUE THREAD

**Issue by Arkweid** [yabeda-rb/yabeda#8]
Prometheus define new settings for metrics `store_setting`
https://github.com/prometheus/client_ruby#aggregation-settings-for-multi-process-stores

It would be nice to have something like this:
```ruby
gauge :users_count do
  comment "Number of total users"
  store aggregation: :max
end
```

Looks like enough to add one more option here:
https://github.com/yabeda-rb/yabeda/blob/master/lib/yabeda/metric.rb
```ruby
option :store,   default: -> { {} }, optional: true, comment: "Custom store settings"
```

**Comment by Envek** [comment]
The only thing that worries me is that it is way too specific for prometheus-client so we need either some general solution or way to specify options only for some adapters.

So, maybe it should be something like this?
```ruby
gauge :users_count do
  adapter :prometheus do
    store_settings aggregation: :max
  end
end
```
I'm not sure yet

**Comment by Envek** [comment]
I've released yabeda 0.5 and yabeda-prometheus 0.5 with support for metric aggregation with following syntax in the Yabeda metric declaration:

```ruby
group :some do
  gauge :tasks do
    comment "Number of test tasks"
    aggregation :max
  end
end
```

Please try it and share your thoughts!
